web/src/app/unite/services/menus.service.ts

```diff
@@ -5,11 +5,13 @@ import { BaseService } from "./base.service";
 @Injectable()
 export class MenusService extends BaseService{
 
+  url: string = 'http://172.132.45.94:3000/api/menus?filter[include][source][extension]=routes';
```

Comment:
This needs to go to a config. You'll need a config that defines the host for the server side component of unite.

Reply:
Changes were done, please check.